build.go

```diff
@@ -19,6 +19,8 @@ import (
 	"github.com/pkg/errors"
 	ignore "github.com/sabhiram/go-gitignore"
 
+	"github.com/buildpacks/lifecycle/platform"
```

Comment:
## Import Groups

Imports should be grouped into the following:

1. Standard library
2. External libraries
3. Local packages